.npmrc

```diff
@@ -0,0 +1 @@
+tag-version-prefix=""
```

Comment:
What issue does this address? `npm` docs say ["only use this property if it is absolutely necessary"](https://docs.npmjs.com/misc/config#tag-version-prefix).

Reply:
This is to remove the leading `v` from git tags created via `npm version`. It is not strictly needed, just for consistency with [existing tags](https://github.com/jsonresume/resume-cli/tags).

If we plan to use the prefixed convention instead, we should remove this.

Reply:
The last tag was created over 4 years ago, so I think it would be reasonable to assume a fresh start that conforms to npms defaults would be okay.

I didn't create those tags so perhaps @thomasdavis could comment.

In any case, I don't think this discussion / change belongs in this branch, so I vote to delete this bit from the branch for this PR.

Reply:
Makes sense, I will drop it. We can always recreate old tags later 🙂

Reply:
Removed.